coap/tracing/adapter.go

```diff
@@ -0,0 +1,68 @@
+package tracing
+
+import (
+	"context"
+
+	"github.com/mainflux/mainflux/coap"
+	"github.com/mainflux/mainflux/pkg/messaging"
+	"github.com/opentracing/opentracing-go"
+)
+
+var _ coap.Service = (*tracingServiceMiddleware)(nil)
+
+const (
+	publish_op     = "coap_publish"
```

Comment:
```suggestion
	publish_op     = "publish_op"
```
No need for coap prefix